### ISSUE TITLE
Notion code blocks still being imported without line breaks

### ISSUE THREAD

**Issue by aksh1618** [obsidianmd/obsidian-importer#107]
Hey there! Notion import has been the main roadblock for me to switch to Obsidian, so thanks for adding this functionality! I tried to import my workspace today, however I'm still facing the issue first reported in #82. I confirmed that the version of the plugin being used is 1.3.1, which should have the fix, however all my code blocks are still ending up without any line breaks (And my notion is absolutely full of them). Let me know if I can provide anything to help debug this further!

![image](https://github.com/obsidianmd/obsidian-importer/assets/26065812/c207dfd5-2220-412d-8422-f8a211f107d0)

_Originally posted by @aksh1618 in https://github.com/obsidianmd/obsidian-importer/issues/82#issuecomment-1717428382_

**Comment by aksh1618** [comment]
I fiddled around today with this issue, and it looks like after replacement, the tags turn into unterminated ones (`<br/>` to `<br>`), which leads to the replacement not working. So first I tried changing the replacement logic to replace `<br>` for code blocks, but that could also fail in edge cases, where the code itself contains `<br>`, which IMO is likely for a code block. So instead I went for adding a pre & post processing using a placeholder, which works without any issues. Submitting a quick MR for the same.
